### DISCUSSION TITLE
参考文献次序以及作者名大写问题

### DISCUSSION THREAD

**Discussion by moontin** [whutug/whu-thesis#236] [Q&A]
大佬们，如何设置参考文献的编号以在文章中出现的次序呢？以及参考文献中作者名全部被转为了大写，这个应该如何修改为bib当中的原始形式呢？

**Comment by sikouhjw** [comment]
> 大佬们，如何设置参考文献的编号以在文章中出现的次序呢？

看一眼手册吧。
![图片](https://github.com/whutug/whu-thesis/assets/49770925/b5451d13-7dfb-437b-85ae-a1d9d765e463)

> 以及参考文献中作者名全部被转为了大写，这个应该如何修改为bib当中的原始形式呢？

https://github.com/sikouhjw/gdutthesis/discussions/68#discussioncomment-2538305

**Reply by moontin** [reply]
请问一下，我这边出现了中文参考文献的作者名称显示不出来的情况，您知道如何解决嘛？
